docs/design/numeric_literals.md

```diff
@@ -0,0 +1,241 @@
+# Numeric Literals
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
```

Comment:
Should make it clear that this doc is not provisional, and reflects all relevant accepted proposals up to that date.

```suggestion

> **STATUS:** Up-to-date on 23-Aug-2022.

```